en/identity-server/7.1.0/docs/apis/restapis/idp.yaml

```diff
@@ -3280,4 +3389,191 @@ components:
+      value:
+        name: "google"
+        description: "IdP for Google Federation"
+        image: "google-logo-url"
+        templateId: "8ea23303-49c0-4253-b81f-82c0fe6fb4a0"
+        isPrimary: false
+        isFederationHub: false
+        homeRealmIdentifier: "localhost"
+        certificate:
+          certificates:
+            - "string"
+          jwksUri: "https://localhost:9444/oauth2/jwks"
+        alias: "https://localhost:9444/oauth2/token"
+        idpIssuerName: "https://www.idp.com"
+        claims:
+          userIdClaim:
+            uri: "http://wso2.org/claims/username"
+          roleClaim:
+            uri: "http://wso2.org/claims/username"
+          mappings:
+            - idpClaim: "country"
+              localClaim:
+                uri: "http://wso2.org/claims/username"
+          provisioningClaims:
+            - claim:
+                uri: "http://wso2.org/claims/username"
+              defaultValue: "sathya"
+        roles:
+          mappings:
+            - idpRole: "google-manager"
+              localRole: "manager"
+          outboundProvisioningRoles:
+            - "manager"
+            - "hr-admin"
+        groups:
+          - name: "google-admin"
+            id: "6b1f8513-3de0-4f28-9cad-b7400dbc94ae"
+        federatedAuthenticators:
+          defaultAuthenticatorId: "U0FNTDJBdXRoZW50aWNhdG9y"
+          authenticators:
+            - authenticatorId: "Y3VzdG9tQXV0aGVudGljYXRvcg"
+              isEnabled: true
+              isDefault: false
+              endpoint:
+                uri: "https://abc.com/token"
+                authentication:
+                  type: "BASIC"
+                  properties:
+                    username: "auth_username"
+                    password: "auth_password"
+        provisioning:
+          jit:
+            isEnabled: true
+            scheme: "PROVISION_SILENTLY"
+            userstore: "PRIMARY"
+            associateLocalUser: true
+            attributeSyncMethod: "OVERRIDE_ALL"
+          outboundConnectors:
+            defaultConnectorId: "U0NJTQ"
+            connectors:
+              - connectorId: "U0NJTQ"
+                isEnabled: true
+                isDefault: false
+                blockingEnabled: false
+                rulesEnabled: false
+                properties:
+                  - key: "somePropertyKey"
+                    value: "somePropertyValue"
+        implicitAssociation:
+          isEnabled: false
+          lookupAttribute:
+            - "email"
+    IdentityProviderWithSystemDefinedAuthenticator:
+      summary: "Request for identity provider with system defined authenticator"
+      value:
+        name: "Custom IDP"
+        description: "IDP with custom federated authenticator"
+        image: "https://custom-authenticator-logo-url"
+        templateId: "8ea23303-49c0-4253-b81f-82c0fe6fb4a0"
+        isPrimary: false
+        isFederationHub: false
+        homeRealmIdentifier: ""
```

Comment:
This is for system defined IdP. Looks like you have swapped the content of two examples